File: 2DAM/ProgramacionMultimedia/Angular/Proyecto3-Aplicacion Biblioteca/ProyectoBiblioteca/src/app/model/libro.ts
export class Libro {
    private _titulo: string;
    private _autor: string;
    private _ISBN: string;
    private _fecha_prestamo: Date;

    constructor(titulo: string, autor: string, ISBN: string, fecha_prestamo: Date) {
        this._titulo = titulo;
        this._autor = autor;
        this. _ISBN = ISBN;
        this._fecha_prestamo = fecha_prestamo;
    }

    get titulo(): string{
        return this._titulo;
    }

    get autor(): string{
        return this._autor;
    }

    get ISBN(): string{
        return this._ISBN;
    }

    get fecha_prestamo(): Date{
        return this._fecha_prestamo;
    }



}

File: 2DAM/Desarrollo De Interfaces/tEMA3-deint/ExamenRecu/ExamenRecuDEINT/ExamenRecuDEINT/Libro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ExamenRecuDEINT
{
    public enum TipoGenero
    {
        LITERARURA,CLASICA,OBRA_MAESTRA
    }
    public class Libro
    {
        public int ISBN { get; set; }
        public string Titulo { get; set; }
        public string Autor {  get; set; }
        public DateTime anoPublicacion { get; set; }
        public int Editorial { get; set; }
        public TipoGenero Genero { get; set; }

        public Libro() { }
        public Libro(int iSBN, string titulo, string autor, DateTime anoPublicacion, int editorial, TipoGenero genero)
        {
            ISBN = iSBN;
            Titulo = titulo;
            Autor = autor;
            this.anoPublicacion = anoPublicacion;
            Editorial = editorial;
            Genero = genero;
        }
    }
}
